typhon-core/src/lib.rs

```diff
@@ -59,10 +60,19 @@ const _: () = {
         fn get() -> &'static Self {
             CELL.get().expect("Settings were not initialized")
         }
-        fn init(settings: Self) {
+        fn init(password: &String) {
+            let password = Box::leak(Box::new(password.clone()));
+            let password = PasswordHash::new(password).expect("Unable to parse the password hash");
+            let settings = Self { password };
```

Comment:
Why not really just move that code to `main` or at least to `init`? if we add fields to settings, things will get repetitive